hip-0000-template.md

```diff
@@ -1,72 +1,491 @@
-created: <date created on>
-discussions-to: <a URL pointing to the official discussion thread>
-updated: <comma separated list of dates>
-requires: <HIP number(s)>
-replaces: <HIP number(s)>
-superseded-by: <HIP number(s)>
+hip: TBD
+**title: Partitioning Functionality for Fungible Tokens on Hedera**
+author: Sam Wood <sam.wood@toko.network>, Stephanie Yi <stephanie.yi@toko.network>, Eleonora Odorizzi <eleonora.odorizzi@toko.network>, Vicky Lio <vicky.lio@toko.network>
+type: Standards Track
+needs-council-approval: Yes
+status: Idea
+created: 2023-06-07
+discussions-to:
+updated: 2023-06-07
 ---
 
 ## Abstract
 
-Please provide a short (~200 word) description of the issue being addressed.
+This HIP proposes a backwards compatible addition of partitioning[^1] functionality to Hedera Fungible Tokens, allowing token issuers to:
+
+* divide and manage tokens into “partitions” (separate groups or bundles);
+* assign unique properties to the partitioned subset, including image URL or other metadata;
+* transfer and manage tokens within each partition independently, effectively making them semi-fungible.
+
+The proposed partitioning feature would address the limitations of fungible tokens, non-fungible tokens, and NFT collections, and would offer greater flexibility, improved compliance, increased scalability, and a better user experience for token issuers across all industries. 
+
 
 ## Motivation
 
-The motivation is critical for HIPs that want to change the Hedera codebase or ecosystem. It should clearly explain why the existing specification is inadequate to address the problem that the HIP solves. HIP submissions without sufficient motivation may be rejected outright.
+The proposed HIP introduces a partitioning feature to effectively and efficiently manage subsets of multiple fungible tokens within a single token ID, meeting business/compliance requirements and reducing overhead costs. 
+
+At present, when referring to fungible supply: 
+
+* Fungible tokens are identical to one another, making it impossible to distinguish tokens within the supply. This limits their use to situations where uniformity and interchangeability are more important than individual token identity.
+* The execution of native primitives[^2] on fungible tokens operates on the entire supply in a uniform manner. As such, any action or operation undertaken on these tokens will impact all other tokens in the supply equally, further emphasizing the indistinguishable nature of fungible tokens.
+
+When referring to non-fungible tokens (NFT) supply: 
+
+* Each NFT has unique properties and metadata. An NFT represents a unique asset or item and cannot be interchanged with other NFTs on the market. 
+* Execution of native primitives<sup>2</sup> affects the single non-fungible tokens. The inability to perform bulk execution of native primitives on multiple NFTs limits scalability due to the computational cost of performing individual operations on a large number of NFTs.
```

Comment:
I could be wrong here, but typically the amount of work done by the network is intentionally proportional to the cost of that work, especially so that no one can tie up the network for free. This doesn't mean bulk operations aren't possible, but it's worth considering how much scaling is acceptable between the two extreme of only one non-fungible token and the other extreme of unlimited non-fungible tokens.

Reply:
we've now removed the discussion around NFTs as it is a digression